wheel/generate_type_stubs.py

```diff
@@ -447,5 +447,18 @@ def derive_child_pk_unhardened(pk: G1Element, index: int) -> G1Element: ...
         ],
     )
 
+    print_class(f, "MerkleSet",
```

Comment:
since this class doesn't derive from streamable or implement any of the special functions the other protocol classes implement, I don't think you should use `print_class()` here. Just print the type stub directly to the output.